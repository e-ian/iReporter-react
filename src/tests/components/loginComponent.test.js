import { shallow } from "enzyme";
import React from "react";
import Login from "../../components/LoginComponent";

describe("login component success tests", () => {
  const props = {
    onSubmit: jest.fn(),
    passwordError: "",
    errors: null,
    generalError: "",
    onPassword: jest.fn(),
    handleChange: jest.fn()
  };
  it("should render without crashing", () => {
    const wrapper = shallow(<Login {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("login component fail tests", () => {
  const props = {
    onSubmit: jest.fn(),
    usernameError: "error",
    passwordError: "error",
    errors: {},
    generalError: "error"
  };
  it("should render without crashing", () => {
    const wrapper = shallow(<Login {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
