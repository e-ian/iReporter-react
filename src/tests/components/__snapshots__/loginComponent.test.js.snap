// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login component fail tests should render without crashing 1`] = `ShallowWrapper {}`;

exports[`login component success tests should render without crashing 1`] = `ShallowWrapper {}`;
